RecoLocalCalo/HGCalRecProducers/plugins/alpaka/HGCalLayerClustersAlgoWrapper.dev.cc

```diff
@@ -38,17 +38,17 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     alpaka::memset(queue, isSeed, 0x0);
 
     algoStandalone.makeClustersCMSSW(size,
```

Comment:
The changes as they are, are correct.

On the other hand one of the advantages of retuning an `std::span` instead of a raw pointer is to be able to pass the size information to a kernel and (optionally) check for out-of-bounds access.

@rovere what would you think about passing the whole views, or at least the idividual spans, to the kernel ?

Reply:
Ciao Andrea, that's something that could be tried, but will requiring some changes also on the external library.
Can we explore this in a separate PR?